tests/Resources/Page/OCPDashboard/Builds/Builds.robot

```diff
@@ -37,4 +37,33 @@ Check Image Build Status
   Run Keyword If    "${build_status}" == "${target_status}"
   ...    Log   The '${build_name}' image match the expected target status
   ...  ELSE
-  ...    Fail  The '${build_name}' image build status doesn't match the target status '${target_status}'
\ No newline at end of file
+  ...    Fail  The '${build_name}' image build status doesn't match the target status '${target_status}'
+
+Get Last Build
+    [Documentation]    Returns last build
+    [Arguments]    ${namespace}    ${word}
+    ${build} =    Run    oc get builds -n ${namespace} | grep ${word} | awk '{print $1;}'
+    [Return]    ${build}
+
+Delete Build Using Name
+    [Documentation]    Delete the build
+    [Arguments]    ${namespace}    ${build_name}
+    Run    oc delete builds ${build_name} -n ${namespace}
```

Comment:
Can be done with the OpenShiftCLI library:
`OpenShiftCLI.Delete  kind=Build  namespace=${namespace}    field_selector=metadata.name==${build_name}`